popup.js

```diff
@@ -0,0 +1,251 @@
+var axios = require("axios");
+var ProgressBar = require("./progressbar.js");
+var Chart = require('chart.js');
+
+document.addEventListener('DOMContentLoaded', function() {
+
+    chrome.tabs.getSelected(null, function(tab) {
+        // Specify your actual API key here:
+        var APIKEY = API_KEY;
+        // Specify the URL you want PageSpeed results for here:
+        var URL_TO_GET_RESULTS_FOR = tab.url;
+        var API_URL = 'https://www.googleapis.com/pagespeedonline/v2/runPagespeed?';
+        var CHART_API_URL = 'http://chart.apis.google.com/chart?';
+
+
+        //event listener for the show suggestions button
+        var suggestionList = document.getElementsByTagName('ul');
+        var suggestionsButton = document.getElementById('suggestions');
+        var hideIt = false;
+        suggestionsButton.addEventListener('click', function() {
+            //toggling between hiding and showing suggestions
+            if (!hideIt) {
+                for (var i = 0; i < suggestionList.length; i++) {
+                    suggestionList[i].style.display = 'block';
+                }
+                hideIt = true;
+            } else {
+                for (var i = 0; i < suggestionList.length; i++) {
+                    suggestionList[i].style.display = 'none';
+                }
+                hideIt = false;
+            }
+        });
+
+
+
+        var graph = document.getElementsByTagName('img');
+        var hideGraph = false;
+        var breakdownButton = document.getElementById('breakdown');
+        breakdownButton.addEventListener('click', function() {
+            if (!hideGraph) {
+                graph[1].style.display = 'block';
+                hideGraph = !hideGraph;
+            } else {
+                graph[1].style.display = 'none';
+                hideGraph = !hideGraph;
+            }
+        });
+
+
+
+
+        //function that removes loading gif 
+        function doneLoading() {
+            var hide = document.getElementsByClassName('loading');
+            hide[0].style.display = 'none';
+            var loadStatus = document.getElementsByClassName('status');
+            loadStatus[0].style.display = 'none';
+            var prgbr = document.getElementById('container');
+            prgbr.display = 'block';
+        }
+
+
+
+
+
+
+
+        function initBar() {
+            var progress = document.getElementById('container');
+            progress.style.display = 'block';
+        }
+
+        function initSuggestion() {
+            var butt = document.getElementById('suggestions');
```

Comment:
got lazy.... didn't want to type button. Also didn't think about the fact that other people would be reviewing my code... I'll chnge it